### PR TITLE
Feature/request access to monitors

### PR DESCRIPTION
**Pull request by claireriley**
Hi folks! This is a PR for Monitoring 1.9 - Request for Access to Monitors. It didn't fit on any existing page, and I noted the gap in docs for editing a monitor, so I created a new page to focus on this. 

@charlie-postman I was unable to get screenshots for the two modals noted in the doc as I don't have the figma permissions to remove arrows and the new feature is not showing up yet in my web or desktop version. Could you please provide these?

### CODE REVIEW

File: src/pages/docs/designing-and-developing-your-api/monitoring-your-api/editing-monitors.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+* [Editing monitors](#editing-monitors)
+
+* [Next steps](#next-steps)
+
+## Requesting access to monitors
+
+Editing a monitor requires you to have edit access for that specific monitor.
+
+In order to get edit access for a monitor, you will need to first be a member of the team the monitor is owned by and a member of the workspace that contains the monitor. Once you have joined the team and [joined the workspace](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#joining-workspaces), navigate to the monitor in question in Postman. In the left sidebar, hover over the monitor and select **...** > **Manage Roles**.
```
Comment (sowmyaraj92) [inline src/pages/docs/designing-and-developing-your-api/monitoring-your-api/editing-monitors.md:37]: **[joined the workspace]** link redirects to `Removing elements from a workspace` section. Can you cross-check on this?

Comment (sowmyaraj92) [inline src/pages/docs/designing-and-developing-your-api/monitoring-your-api/editing-monitors.md:37]: I would consider rephrasing the first line — "To get edit access for a monitor, you will first need to be a member of the team that owns the monitor, and then a member of the workspace that contains the monitor."

**Review by sowmyaraj92 — COMMENTED** [review]
@claireriley This looks good, suggested minor changes. Once the screenshots are updated, we should be good to merge.
Since it's a new page, please do make sure there are no broken links in "redirects.json".

**Comment by charlie-postman** [comment]
Hi Claire, 

Thanks for consolidating the page. Is this page (https://learning.postman.com/docs/collaborating-in-postman/requesting-access-to-collections/) a better place for the documentation? We can put something together similar to RFA for Collections and APIs.

**Pull request closed without merging**